windows/include/citus_version.h

```diff
@@ -0,0 +1,26 @@
+/* Citus full name as a string */
```

Comment:
Why is this PR changing this?

Reply:
Similar story. `citus_version.h` is generated by our Makefile. On Windows we don't use Make, so we have to hard-code it. This is very suboptimal but hard to avoid.